src/StackEmptyException.java

```diff
@@ -0,0 +1,12 @@
+public class StackEmptyException extends Throwable {
+
+	/**
+	 * 
+	 */
+	private static final long serialVersionUID = 8940747433345773938L;
+
+	public StackEmptyException() {
+		System.out.println("Stack is empty");
```

Comment:
Would you normally want an exception to include functionality? I thought they were mostly concerned with detailing information about whatever caused the exception and handling them was left to catch blocks.